src/resources/js/v2/tickets-commerce.js

```diff
@@ -200,7 +184,7 @@ tribe.tickets.commerce = {};
 			return purchaser;
 		}
 
-		purchaser.name  = $container.find( obj.selectors.purchaserName ).val();
+		purchaser.name = $container.find( obj.selectors.purchaserName ).val();
 		purchaser.email = $container.find( obj.selectors.purchaserEmail ).val();
```

Comment:
In this case, we're aligning the equals signs with the `purchaser.email` line just so it's easier to read.

Reply:
Sure, but it wasn't a change I just made randomly. It was noted by eslint for extra spaces.

Reply:
Oh... In that case, never mind. LOL